docker/dockerfiles_update_date.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# exit on errors
+set -e
+
+#
+# Will go over all Dockerfiles that have a comment of the form: 
```

Comment:
will this script manually or automatically? Because I don't see who triggers this script

Reply:
It is a utility script. To run manually when you need to update the python images as I just saw that is needed. In the future need to see how we do this automatically.